src/services/fileServices/MdPageServices/MediaFileService.js

```diff
@@ -21,7 +21,18 @@ class MediaFileService {
 
   async create(sessionData, { fileName, directoryName, content }) {
     this.mediaNameChecks({ directoryName, fileName })
-    const sanitizedContent = await validateAndSanitizeFileUpload(content)
+
+    const [schema, fileContent] = content.split(",")
```

Comment:
could we add a comment on why this is required and what the `schema` is? separately, this destructuring assignment always takes the second element as the `fileContent` - are we guaranteed that this is so?

Reply:
Hey @seaerchin , this line is actually not written by me. It was in the original code, since my file virus scan also needs the fileContent, so I abstracted this line out of the `validateAndSanitizeFileUpload`  function to avoid duplicate code. 
My guess is it splits the incoming content stream by a comma, and the second part would always be the actual file content (encoded base64 string) since it's a static split. 
Not sure what the `schema` is, might be the algorithm used to encode the file. @alexanderleegs do you know what the `schema` is?

Reply:
Sure, that sounds good. The unused declaration should be removed. I fixed it in 47bba00269aa4b540de793007d6952bf621b2d1b

@alexanderleegs fileContent is the intended one and scheme is never used.